galgebra/mv.py

```diff
@@ -2132,7 +2129,7 @@ def Add(dop1, dop2):
                 raise ValueError('In Dop.Add Dop arguments are not from same geometric algebra')
 
             if dop1.cmpflg != dop2.cmpflg:
-                raise ValueError('In Dop.Add complement flags have different values.')
+                raise ValueError('In Dop.Add complement flags have different values: %s vs. %s' % (dop1.cmpflg, dop2.cmpflg))
```

Comment:
cmpflg still has some potential bugs. This is to make the exception to have proper context.